### ISSUE TITLE
Source tabs in docs should not be in the tab order when visually hidden

### ISSUE THREAD

**Issue by marcysutton** [angular/material#501]
When navigating through the Material Design demos with the keyboard, tabs hidden under the "Source" button are still tabbable, creating an inaccessible user experience. To be accessible when the source is not enabled, the tabs should be removed from the tab order with either `tabIndex="-1"` or with wrapping CSS of `display: none;`

Bottom Sheet demo where tabs should be really hidden
![Bottom Sheet demo with source closed](https://cloud.githubusercontent.com/assets/1045233/4836577/b879aecc-5fc6-11e4-842b-361c4628d18d.png)

Source enabled, tabs should be in the tab order
![Bottom Sheet demo with source open](https://cloud.githubusercontent.com/assets/1045233/4836589/cd5befb2-5fc6-11e4-8634-04e3609b8bf2.png)
